### DISCUSSION TITLE
TV and Kiosk mode in developer environment

### DISCUSSION THREAD

**Discussion by adeel86** [grafana/grafana#63111] [Dashboards, Panels & Visualizations]
Grafana v9.4.0-pre 

I have created a playlist of dashboard but when I play it then TV and Kiosk mode didn't work. It is just showing the normal mode. I also didn't receive any error on logs. 

I have followed the following tutorial for setting up developer environment and playlist. 
[Developer guide](https://github.com/grafana/grafana/blob/HEAD/contribute/developer-guide.md)
[Manage playlists](https://grafana.com/docs/grafana/latest/dashboards/create-manage-playlists/#:~:text=A%20playlist%20is%20a%20list,them%20perfect%20for%20big%20screens.)

Can anybody tell me if I am missing something which causing this issue?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
